### PR TITLE
feat: disable login button

### PR DESCRIPTION
**Pull request by banahaker**
# Description
Disables the login button to avoid unexpected errors on user login.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,7 +58,13 @@ <h1>Lipoic</h1>
       <div class="right">
         <h1>Lipoic 來品一課</h1>
         <h2>線上整合教育平台</h2>
-        <a class="loginBtn" href="https://app.lipoic.org">登入</a>
+        <!-- <a class="loginBtn" href="https://app.lipoic.org">登入</a> -->
+        <a
+          class="loginBtn"
+          href="#"
+          onclick="alert('應用正在開發中，尚未開放註冊。若您為開發者，欲貢獻開發，請聯繫Lipoic官方(discord)')"
```
Comment (lumynou5) [inline index.html]:
```suggestion
          onclick="alert('應用正在開發中，尚未開放註冊。若您為開發者，欲貢獻開發，請聯繫 Lipoic 官方（Discord）。')"
```
if you want to use Chinese

**Review by lumynou5 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (SiongSng) [inline index.html]:
```suggestion
          onclick="alert('Lipoic 來品一課網頁版正在開發中，目前尚未開放公眾註冊使用，倘若造成您的困擾或不便，我們深表歉意。\n若您想參與 Lipoic 的開發，麻煩您透過 Discord 等方式聯繫我們（詳見本站底部），謝謝！')"
```

**Review by SiongSng — CHANGES_REQUESTED** [review]
我改了一下說明的敘述，這樣子更完整與正式一點。

File: index.html [hunk 2]
```diff
@@ -58,7 +58,13 @@ <h1>Lipoic</h1>
       <div class="right">
         <h1>Lipoic 來品一課</h1>
         <h2>線上整合教育平台</h2>
-        <a class="loginBtn" href="https://app.lipoic.org">登入</a>
+        <!-- <a class="loginBtn" href="https://app.lipoic.org">登入</a> -->
+        <a
+          class="loginBtn"
+          href="#"
+          onclick="alert('應用正在開發中，尚未開放註冊。若您為開發者，欲貢獻開發，請聯繫 Lipoic 官方（Discord）。')"
```
Comment (SiongSng) [inline index.html]:
```suggestion
          onclick="alert('Lipoic 來品一課網頁版正在開發中，目前尚未開放公眾註冊使用，倘若造成您的困擾或不便，我們深表歉意。\n若您想參與 Lipoic 的開發，麻煩您透過 Discord 等方式聯繫我們（詳見本站底部），謝謝。')"
```

**Review by SiongSng — CHANGES_REQUESTED** [review]

**Review by SiongSng — APPROVED** [review]
LGTM

**Review by lumynou5 — APPROVED** [review]

**Pull request merged**